tools/spectral/functions/acceptHeaderUpcomingVersionLimit.js

```diff
@@ -0,0 +1,41 @@
+module.exports = function (input) {
```

Comment:
can we have unit tests like we have for ipa rule sets?

https://github.com/mongodb/openapi/tree/main/tools/spectral/ipa/__tests__

Reply:
I missed those, didn't know it was possible! Thanks